content/_index/coffeeeditorlinks/tryitnow.md

```diff
@@ -0,0 +1,10 @@
++++
+title = "Try it now!"
+weight = 10
+
+[asset]
+  icon = "fas fa-play"
+  url = "http://35.234.104.175:9091/services/launch"
```

Comment:
@jfaltermeier can we use a different url eg eclipsesource.com/coffee-editor/try , I find the IP not very trustworthy ;-)

Reply:
@eneufeld As a first improvement I'll create an HTTP redirect from http://coffee-editor.eclipsesource-munich.com/ 

I created it, but at least on my machine it is not propagated yet.

Reply:
@jfaltermeier works for me, thank you.
@JonasHelming can you update the url?

Reply:
done